### PR TITLE
fix: coler picker support touch

### PR DESCRIPTION
**Pull request by kooriookami**
Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: packages/components/color-picker/src/components/alpha-slider.vue [hunk 1]
```diff
@@ -109,12 +109,32 @@ export default defineComponent({
       }
     }
 
+    const getClientXY = (event: MouseEvent | TouchEvent) => {
```
Comment (btea) [inline packages/components/color-picker/src/components/alpha-slider.vue:112]: This method is used in several places. May be is it better to extract it alone.

Comment (kooriookami) [reply packages/components/color-picker/src/components/alpha-slider.vue]: You are right, I added `getClientXY` as a util.

File: packages/utils/util.ts [hunk 2]
```diff
@@ -248,3 +248,22 @@ export const refAttacher = <T extends HTMLElement | ComponentPublicInstance>(
     ref.value = val
   }
 }
+
+export const getClientXY = (event: MouseEvent | TouchEvent) => {
```
Comment (sxzz) [inline packages/utils/util.ts:252]: We can move this function to `dom.ts`

Comment (kooriookami) [reply packages/utils/util.ts:252]: OK~

**Review by btea — APPROVED** [review]

**Review by jw-foss — APPROVED** [review]

**Review by jw-foss — APPROVED** [review]

**Pull request merged**